.eslintrc.js

```diff
@@ -83,6 +83,7 @@ module.exports = {
     '@typescript-eslint/no-shadow': 'error',
     '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
     '@typescript-eslint/no-use-before-define': ['error'],
+    'camelcase': 'off'
```

Comment:
@gutakk We shouldn't turn off this rule completely, If there's a specific file that is conflicting with this rule, then we can turn off this rule for that specific file.

Reply:
Agree with you. Fixed in a740de03bb3838afcdbd51e67f7d384ba55b33d3